### PR TITLE
Proofreading - Using Version Control in Studio Pro

### PR DESCRIPTION
**Pull request by Yiyun333**

### CODE REVIEW

File: content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md [hunk 1]
```diff
@@ -132,91 +135,89 @@ To push changes, select **Version Control** > **Push** or make sure that the **A
...
 
 {{< figure src="/attachments/refguide/version-control/using-version-control-in-studio-pro/main-branch-line.png" >}}
 
-It is often convenient to have more than one development line. For example, one development line for fixing bugs in the currently deployed version of your app and another line where you develop new functionality. If you then find a bug in the deployed version you can fix it in the corresponding development line irrespective of the state of the development line where new functionality is developed. See [Version Control](/refguide/version-control/) for more information about branches.
+It is often convenient to have more than one development line. For example, one development line is for fixing bugs in the currently deployed version of your app and another line is where you develop new functionality. If you then find a bug in the deployed version, you can fix it in the corresponding development line irrespective of the state of the development line where new functionality is developed. For more information about branches, see [Version Control](/refguide/version-control/).
```
Comment (Yiyun333) [inline content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md]: I suggest to change the cross-reference to: see the [Branches](/refguide/version-control/#branches) section in *Version Control*.

File: content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md [hunk 2]
```diff
@@ -132,91 +135,89 @@ To push changes, select **Version Control** > **Push** or make sure that the **A
...
 
-Usually, combining your changes with the latest revision from the repository will be done automatically. For example, one person may add a page while you are changing a microflow. If the changes are too close, however, a conflict can arise. For example, if one of your team has changed the properties of the same data view which you have also changed. You will have to resolve such conflicts before you can commit. See the  [Dealing With Conflicts](#conflicts) section, below, for information on how to do this.
+Usually, combining your changes with the latest revision from the repository is done automatically. For example, one person may add a page while you are changing a microflow. If the changes are too close, however, a conflict can arise. For example, if one of your team has changed the properties of the same data view that you have also changed, you will have to resolve such conflicts before you can commit. For information on how to do this, see the [Dealing With Conflicts](#conflicts) section below.
```
Comment (Yiyun333) [inline content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md]: I suggest linking the cross-reference to the [Combining Changes and Conflict Resolution](https://docs.mendix.com/refguide/merge-algorithm/) doc.

So, I will use the following for the cross-reference:

> For information on how to do this, see [Combining Changes and Conflict Resolution](/refguide/merge-algorithm/).

File: content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md [hunk 3]
```diff
@@ -132,91 +135,89 @@ To push changes, select **Version Control** > **Push** or make sure that the **A
...
 
 To update the working copy of your app, click **Pull** in the **Changes** pane, or choose the **Version Control** > **Pull** menu item.
 
-{{< figure src="/attachments/refguide/version-control/using-version-control-in-studio-pro/update-button.png" alt="Update Button" >}}
+{{< figure src="/attachments/refguide/version-control/using-version-control-in-studio-pro/pull-button.png" max-width=60% class="image-border" alt="Pull Button" >}}
 
 Changes you receive from the repository when pulling are combined with the changes you have made to your working copy (if any). Afterwards, your working copy will contain both your changes and the changes you received. As part of the pull, the original of your working copy is updated as well.
 
 For example, if the last time you pulled you received all changes up to and including revision N, this means that the original for your working copy is revision 40. Since you started making changes to your working copy, other people on your team have made another four commits (X, Y, and Z). If you now pull, you will receive those changes and Z will be the new *original* to which your changes are compared.
```
Comment (Yiyun333) [inline content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md]: There are two places in this paragraph that are confusing to me:

- **revision 40** - is it **revision N** according to the context?
- **four commits** - in the parentheses, there are only X, Y, Z (so, it should be **three commits**?)

File: content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md [hunk 4]
```diff
@@ -366,41 +367,39 @@ If you are using a different hosting environment, you create a deployment packag
...
+If you add or delete files (or directories) using Windows Explorer, Studio Pro automatically adds or deletes these from version control too.
 
 Make sure you use the **Export** feature of TortoiseGit if you are copying a directory that is already under version control in your app.
 
 ### 9.4 Deleting Files and Directories
 
-If you delete a file from your app, Studio Pro will automatically also delete it from the Team Server.
+If you delete a file from your app, Studio Pro automatically deletes it from the Team Server too.
 
 A folder is no longer be tracked if all the files in the folder are removed.
```
Comment (Yiyun333) [inline content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md]: The heading of section 9.3 only mentions adding files and directories. However, the first paragraph of this section mentions deleting files too. 

It seems that section 9.4 is about deleting files and directories. However, the content of the two sections does not look the same and uses different terminology (for example., **delete files (or directories) using Windows Explorer**, **delete a file from your app**). So, I am also not sure whether I should delete **delete files (or directories)** from section 9.3.

File: content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md [hunk 5]
```diff
@@ -255,7 +256,7 @@ You can delete the branch after merging it back, if you want.
 
 #### 7.2.1 Branching
 
-Development lines other than the main line are called branch lines. Our advice would be to develop new features in the *main line* and to use *branch lines* for fixing bugs in versions that have been deployed. This is the scenario Studio Pro makes easy but other scenarios for more complex apps are supported as well.
+Development lines other than the main line are called branch lines. Our advice would be to develop new features in the main line and to use branch lines for fixing bugs in versions that have been deployed. This is the scenario Studio Pro makes easy but other scenarios for more complex apps are supported as well.
 
 You can create branch lines from the Branch Line Manager which you can find at **Version Control > Manage Branch Lines...**.
```
Comment (Yiyun333) [inline content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md]: This instruction is almost the same as the one on line 238, but the screenshots show different things. 

I was wondering whether we could merge section 7.1 and section 7.2 in some way. The content of the first two sub-sections of section 7.2 seems to overlap with the content in 7.1.2 somehow with some inconsistency and different emphases, which can be very confusing. Also, if we merge these sections, we can start section 7 using the header **Working with Branches in Studio Pro**, which is much easier to understand and navigate, I think :)

Btw, I also suggest providing a cross-reference to this section (section 7) in the [Branches](https://docs.mendix.com/refguide/version-control/#branches) section in _Version Control_.

File: content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md (hunk 4 above)
Comment (MariaShaposhnikova) [reply content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md]: Good catch, I will delete 9.4, but will double-check if I deleted any valuable info.

File: content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md (hunk 5 above)
Comment (MariaShaposhnikova) [reply content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md]: @Yiyun333 , thank you for your suggestion. I have made changes in this commit: https://github.com/mendix/docs/pull/7449/commits/b8a516a28ccf06d02f968804deb966a0c57e19ca

Comment (Yiyun333) [reply content/en/docs/refguide/version-control/using-version-control-in-studio-pro/_index.md]: The changes look great! Thanks for the effort and the re-structuring!

**Pull request merged**